packages/react/src/components/ComboBox/ComboBox.stories.js

```diff
@@ -106,9 +103,126 @@ export const WithLayer = (args) => (
           placeholder="Filter..."
           titleText="Third Layer"
           helperText="Combobox helper text"
-          {...args}
         />
       </Layer>
     </Layer>
   </div>
 );
+
+export const Playground = (args) => (
+  <div style={{ width: 300 }}>
+    <ComboBox
+      onChange={() => {}}
+      id="carbon-combobox"
+      items={items}
+      downshiftProps={{
+        onStateChange: () => {
+          console.log('the state has changed');
+        },
+      }}
+      itemToString={(item) => (item ? item.text : '')}
+      placeholder="Filter..."
+      titleText="ComboBox title"
+      helperText="Combobox helper text"
+      {...args}
+    />
+  </div>
+);
+
+Playground.argTypes = {
+  ariaLabel: {
+    table: {
+      disable: true,
+    },
+  },
+  id: {
+    table: {
+      disable: true,
+    },
+  },
+  downshiftProps: {
+    table: {
+      disable: true,
+    },
+  },
+  initialSelectedItem: {
+    table: {
+      disable: true,
+    },
+  },
+  invalidText: {
+    table: {
+      disable: true,
+    },
+  },
+  items: {
+    table: {
+      disable: true,
+    },
+  },
+  itemToElement: {
+    table: {
+      disable: true,
+    },
+  },
+  itemToString: {
+    table: {
+      disable: true,
+    },
+  },
+  light: {
+    table: {
+      disable: true,
+    },
+  },
+  onChange: {
+    table: {
+      disable: true,
```

Comment:
For all the callback props, I think if you define them in the component level argTypes up top they'll get logged out to the `Actions` panel

```js
argTypes: { onChange: { action: 'clicked' } },
```